modules/custom-status/custom-status.php

```diff
@@ -459,747 +466,806 @@ function post_admin_header() {
-			die( $change_error->get_error_message() );
-		}
+			// Check to make sure the status doesn't already exist
+			if ( $this->get_custom_status_by( 'slug', $status_slug ) && ( $this->get_custom_status_by( 'id', $term_id )->slug != $status_slug ) ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Status already exists. Please choose another name.', 'edit-flow' ) );
+				die( esc_html( $change_error->get_error_message() ) );
+			}
 
-		// Check to make sure the status doesn't already exist as another term because otherwise we'd get a fatal error
-		$term_exists = term_exists( sanitize_title( $status_name ), self::taxonomy_key );
-		if ( $term_exists && isset( $term_exists['term_id'] ) && $term_exists['term_id'] != $term_id ) {
-			$change_error = new WP_Error( 'invalid', __( 'Status name conflicts with existing term. Please choose another.', 'edit-flow' ) );
-			die( $change_error->get_error_message() );
-		}
+			// Check to make sure the status doesn't already exist as another term because otherwise we'd get a fatal error
+			$term_exists = term_exists( sanitize_title( $status_name ), self::taxonomy_key );
+			if ( $term_exists && isset( $term_exists['term_id'] ) && $term_exists['term_id'] != $term_id ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Status name conflicts with existing term. Please choose another.', 'edit-flow' ) );
+				die( esc_html( $change_error->get_error_message() ) );
+			}
 
-		// get status_name & status_description
-		$args = array(
-			'name' => $status_name,
-			'description' => $status_description,
-			'slug' => $status_slug,
-		);
-		$return = $this->update_custom_status( $term_id, $args );
-		if( !is_wp_error( $return ) ) {
-			set_current_screen( 'edit-custom-status' );
-			$wp_list_table = new EF_Custom_Status_List_Table();
-			$wp_list_table->prepare_items();
-			echo $wp_list_table->single_row( $return );
-			die();
-		} else {
-			$change_error = new WP_Error( 'invalid', sprintf( __( 'Could not update the status: <strong>%s</strong>', 'edit-flow' ), $status_name ) );
-			die( $change_error->get_error_message() );
+			// get status_name & status_description
+			$args = array(
+				'name' => $status_name,
+				'description' => $status_description,
+				'slug' => $status_slug,
+			);
+			$return = $this->update_custom_status( $term_id, $args );
+			if ( ! is_wp_error( $return ) ) {
+				set_current_screen( 'edit-custom-status' );
+				$wp_list_table = new EF_Custom_Status_List_Table();
+				$wp_list_table->prepare_items();
+				echo wp_kses_post( $wp_list_table->single_row( $return ) );
+				die();
+			} else {
+				/* translators: 1: the status's name */
+				$change_error = new WP_Error( 'invalid', wp_kses( sprintf( __( 'Could not update the status: <strong>%s</strong>', 'edit-flow' ), $status_name ), 'strong' ) );
+				die( esc_html( $change_error->get_error_message() ) );
```

Comment:
We're double escaping the `<strong></strong>` here. The Moving the `wp_kses()` call into die() should fix that:

```suggestion
				die( wp_kses( $change_error->get_error_message(), 'strong' ) );
```

Reply:
Good catch, the linter complained about both of em and I ended up messing up the right escape to use.

Reply:
Done